build_projects/dotnet-host-build/project.json

```diff
@@ -6,7 +6,7 @@
   },
   "dependencies": {
     "NETStandard.Library": "1.6.0-rc3-24201-00",
-    "Microsoft.NETCore.Runtime": "1.0.2-rc3-24201-00",
+    "Microsoft.NETCore.Runtime.CoreCLR": "1.0.2-rc3-24201-00",
```

Comment:
A previously built one there's no explicit dependency on latest here, I can see the difficulty with that for update dependencies though, it's a bootstrapping loop